pootle/apps/reports/views.py

```diff
@@ -204,6 +204,12 @@ def get_detailed_report_context(user, month):
                 else:
                     totals['reviewed'][score.review_rate] = {'words': reviewed}
 
+            suggested = score.get_suggested_words()
+            if suggested is not None:
+                score.action = _('Suggestion')
```

Comment:
What is the main concern? Don't mutate `score` object or feed a dict instead of an object to template? Can it be a dict like this `{'action': 'XXX', 'score': score,...}`?